src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    public static final int MAXVALUE = 100;

```

Comment:
```suggestion
    private static final int MAXVALUE = 100;

```
якщо змінна не використовується поза межами класу краще зробити її приватною